admin/release.py

```diff
@@ -409,7 +409,12 @@ def publish_homebrew_recipe(homebrew_repo_url, version, source_bucket,
     # Sometimes this raises an index error, and it seems to be a race
     # condition. There should probably be a loop until push succeeds or
     # whatever condition is necessary for it to succeed is met. FLOC-2043.
-    push_info = homebrew_repo.remotes.origin.push(homebrew_repo.head)[0]
+    push_info = homebrew_repo.remotes.origin.push(
+        homebrew_repo.head,
+        # Ignore any hooks which might prevent pushing (to master in this
+        # case). Without this, the release process can hang.
+        no_verify=True,
```

Comment:
There are three options for this wrt testing:
* Convince `git` to use some particular fake hook in the test, test that the code in the hook is not run,
* Use a spy for push to check that the `no_verify` option is used
* Don't test this

I went for the latter as a judgement call of the tradeoff between effort and value